content/en/docs/reference/kubectl/_index.md

```diff
@@ -159,66 +159,66 @@ To learn more about command operations, see the [kubectl](/docs/reference/kubect
 
 The following table includes a list of all the supported resource types and their abbreviated aliases.
 
-(This output can be retrieved from `kubectl api-resources`, and was accurate as of Kubernetes 1.19.1.)
+(This output can be retrieved from `kubectl api-resources`, and was accurate as of Kubernetes 1.25.0)
 
-| NAME | SHORTNAMES | APIGROUP | NAMESPACED | KIND |
+| NAME | SHORTNAMES | APIVERSION | NAMESPACED | KIND |
```

Comment:
Aside: these headings would work better as full words in sentence case (“Name”, “API group”, etc).